server/models/Pt.js

```diff
@@ -21,7 +22,7 @@ const PtSchema = new Schema(
       type: Number,
     },
     location: {
-      type: String,
+      type: PointSchema,
```

Comment:
what is a pointSchema?

Reply:
It’s used so that we can use geojson to provide location coordinates.